app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt

```diff
@@ -75,35 +75,50 @@ fun HomeDrawer(
         Logo()
 
         topItems.forEach { item ->
-            DrawerItem(
-                data = item.getDrawerData(),
-                selected = currentRoute == item.route,
-                onItemClick = {
-                    navigateToItemInHome(homeNavController, item)
-                    scope.launch { drawerState.close() }
-                }
-            )
+            if (item != HomeNavigationItem.Settings)
+                DrawerItem(
+                    data = item.getDrawerData(),
+                    selected = currentRoute == item.route,
+                    onItemClick = {
+                        navigateToItemInHome(homeNavController, item)
+                        scope.launch { drawerState.close() }
+                    }
+                )
         }
 
         Spacer(modifier = Modifier.weight(1f))
 
-        val bottomItems = listOf(Settings, Support, Debug)
+        val bottomItems = listOf(HomeNavigationItem.Settings, Support, Debug)
         bottomItems.forEach { item ->
             DrawerItem(
```

Comment:
Suggestion: 

```
bottomItems.forEach { item ->
    val (isSelected, onClick: () -> Unit) = when (item) {
        is HomeNavigationItem -> (currentRoute == item.route) to {
            navigateToItemInHome(homeNavController, item)
            closeDrawer()
            Unit
        }
        is NavigationItem -> false to {
            when (item.isExternalRoute()) {
                true -> CustomTabsHelper.launchUrl(homeNavController.context, item.getRouteWithArgs())
                false -> viewModel.navigateTo(item)
            }
            closeDrawer()
            Unit
        }
        else -> false to {}
    }

    DrawerItem(
        data = item.getDrawerData(),
        selected = isSelected,
        onItemClick = onClick
    )
}
```
the best thing about it is that we'll have only 1 check `item == HomeNavigationItem` instead of 2 for `selected` and `onItemClick`

also somewhere at the top create `val closeDrawer = { scope.launch { drawerState.close() } }` and re-use it everywhere.

And in `HomeViewModel`
```
fun navigateTo(item: NavigationItem) {
   viewModelScope.launch { navigationManager.navigate(NavigationCommand(destination = item.getRouteWithArgs())) }
}

fun navigateToUserProfile() = navigateTo(NavigationItem.SelfUserProfile)
```